src/Log/logListener.ts

```diff
@@ -52,6 +58,8 @@ const addLogEntriesToStore = (dispatch: TDispatch) => () => {
     }
 };
 
+const formatNrfdlLogs = (msg: string) => msg.split(' ').slice(1).join(' ');
```

Comment:
If I understand this correctly, this essentially cuts of everything before the first space. Why is this done?

Reply:
Used that as a placeholder to remove the timestamp. Will keep the timestamp though and instead set a proper log pattern.